inference-engine/tests/functional/plugin/gna/pass_tests/fq_fusion_with_sigmoid.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ie_core.hpp>
+#include "ngraph_functions/builders.hpp"
+#include "common_test_utils/test_constants.hpp"
+#include "shared_test_classes/base/layer_test_utils.hpp"
+
+namespace LayerTestsDefinitions {
+
+typedef std::tuple<
+    std::string,                        // Target device name
+    InferenceEngine::Precision,         // Network precision
+    size_t,                             // Input size
+    std::map<std::string, std::string>  // Configuration
+> fqFusionWithSigmoidParams;
+
+class FqFusionWithSigmoidTest : public LayerTestsUtils::LayerTestsCommon,
+    public testing::WithParamInterface<fqFusionWithSigmoidParams> {
+protected:
+    void SetUp() override {
+        InferenceEngine::Precision netPrecision;
```

Comment:
do we really need 3 fake quantize layers here to reproduce the problem case? or just one after sigmoid?

Reply:
Two unnecessary FQ layers were removed.